tslearn/docs/dtw.rst

```diff
@@ -85,6 +85,30 @@ simplicity):
        return sqrt(C[n, m])
 
 
+Using a different ground metric
```

Comment:
I took the freedom to change this page so that the paragraph is included earlier, which looks more natural, I think.

Reply:
Looks better indeed !